src/state/helpers.ts

```diff
@@ -17,6 +17,7 @@ export const isCrossAccountTradeSupported = (swapperName: SwapperName) => {
     case SwapperName.ArbitrumBridge:
     case SwapperName.Portals:
     case SwapperName.Test:
+    case SwapperName.ButterSwap:
```

Comment:
It *is* supported, looks like the reason why I wasn't able to get native quotes

Reply:
?

Reply:
f242f27b8c